metaflow/plugins/kubernetes/kubernetes_cli.py

```diff
@@ -275,6 +275,16 @@ def _sync_metadata():
                 ),
             )
 
+    env.update(
```

Comment:
Can you add a note here why these are needed so that they are not wiped by mistake.